### PR TITLE
Added some new terms into the glossary

### PR DESCRIPTION
**Pull request by ntounsi**
ADDED to Glossary
Mabsut with definition
Mukawwar with definition
Abreviation 
Connexion / disconnection 

Ligature       new alternate term 
Baseline      new alternate term

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4983,6 +4990,14 @@
           </td>
         </tr>
 
+      <tr id="def_connection(orconnexion)">
```
Comment (mostafah) [inline index.html]: No need for parenthesis in the id.

File: index.html [hunk 2]
```diff
@@ -4139,6 +4139,14 @@
 
 
       <tbody>
+      <tr id="def_abreviation">
+        <td>abreviation</td>
```
Comment (r12a) [inline index.html]: Abbreviation needs to have two b's.

File: index.html (hunk 1 above)
Comment (r12a) [reply index.html]: Connexion is an incorrect spelling for connection.

File: index.html [hunk 3]
```diff
@@ -6987,7 +7015,15 @@
           </td>
         </tr>
 
-
+      <tr id="def_Mukawwar">
+        <td>Mukawwar</td>
```
Comment (mostafah) [inline index.html]: Like our other words, this should probably be in lower case. (“mukawwar” instead of “Mukawwar”.)

**Comment by behnam** [comment]
Review notes from the meeting:
* Use all-lowercase for terms (except for proper nouns), so "Mukawwar" would be "mukawwar".
* No need to have "connexion" in the terms.
* Update description of "mabsut" and "mukawwar" to something shorter, simpler and more direct, like what's there for Bast and Takweer.

**Comment by ntounsi** [comment]
Done ACTION-91: Update pull request 114 with simpler definitions

**Comment by ntounsi** [comment]
Done ACTION-91: Update pull request 114 with simpler definitions

**Comment by behnam** [comment]
Thanks, @ntounsi, for the update!

**Pull request merged**